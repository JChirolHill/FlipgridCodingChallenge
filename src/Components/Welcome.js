export default function Welcome(props) {
    return (
        <>
            <div data-testid="email">{props.email}</div>

            <div className="text-right">
                <button type="button" className="btn btn-fg font-weight-bold">Sign In</button>
            </div>
        </>
    );
}