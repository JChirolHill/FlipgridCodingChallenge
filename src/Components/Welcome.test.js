import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Welcome from './Welcome.js';

const email = 'hello@flipgrid.com';

afterEach(cleanup);

it('renders', () => {
    const { asFragment } = render(<Welcome email={email}/>);

    expect(asFragment()).toMatchSnapshot();
});

it('puts email in email div', () => {
    const { getByTestId } = render(<Welcome email={email}/>);

    expect(getByTestId('email')).toHaveTextContent(email);
});