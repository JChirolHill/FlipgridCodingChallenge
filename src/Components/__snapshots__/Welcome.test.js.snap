// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    data-testid="email"
  >
    hello@flipgrid.com
  </div>
  <div
    class="text-right"
  >
    <button
      class="btn btn-fg font-weight-bold"
      type="button"
    >
      Sign In
    </button>
  </div>
</DocumentFragment>
`;
